### PR TITLE
Add check that status with listener is not gc'd

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: permissions/all-permissions.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    "gyroscope",
+    "magnetometer",
+    "midi",
+    "nfc",
+    "screen-wake-lock",
+    "camera",
+    "display-capture",
+    "microphone",
+    "notifications",
+    "speaker-selection",
```
Comment (miketaylr) [inline permissions/all-permissions.html:25]:
```suggestion
    "microphone",
    "speaker-selection",
```

Comment (marcoscaceres) [reply permissions/all-permissions.html]: Oopd

**Review by miketaylr — APPROVED** [review]
lgtm!

**Comment by marcoscaceres** [comment]
Sorry, I’ll clean this up.

**Comment by miketaylr** [comment]
Or, I will break the rules and just merge. Don't call the cops.

**Comment by marcoscaceres** [comment]
Ha! Good call.

**Pull request merged**